images/linux/toolsets/toolset-1804.json

```diff
@@ -109,7 +109,7 @@
             "name": "az",
             "url" : "https://raw.githubusercontent.com/Azure/az-ps-module-versions/main/versions-manifest.json",
             "versions": [
-                "5.9.0"
```

Comment:
as we are upgrading the major version from 5 => 6, lets zip the 5.9.0 so that in case there is any breaking change, customer can get unblocked quickly using 5.9.0 as hard coded task input and will not see any performance issue